src/sentry/web/frontend/organization_integration_setup.py

```diff
@@ -42,11 +42,13 @@ def handle(self, request: Request, organization, provider_id) -> HttpResponseBas
             is_feature_enabled = {}
             for feature in pipeline.provider.features:
                 feature_flag_name = "organizations:integrations-%s" % feature.value
-                is_feature_enabled[feature_flag_name] = True
-                if feature_flag_name in settings.SENTRY_FEATURES:
+                try:
+                    features.get(feature_flag_name, None)
                     is_feature_enabled[feature_flag_name] = features.has(
```

Comment:
Kind of annoying that `features.has` doesn't allow us to specify a default on error